src/nl/surf/eduhub_rio_mapper/processing.clj

```diff
@@ -115,11 +115,11 @@
                            :as          job} :job
                           mutate-result      :mutate-result
                           eduspec            :eduspec}]
-    (let [rio-code (when-not (blocking-retry #(resolver type id institution-oin)
-                                             (:rio-retry-attempts-seconds rio-config)
-                                             "Ensure upsert is processed by RIO")
-                     (throw (ex-info (str "Processing this job takes longer than expected. Our developers have been informed and will contact DUO. Please try again in a few hours."
-                                          ": " type " " id) {})))]
+    (let [rio-code (or (blocking-retry #(resolver type id institution-oin)
```

Comment:
Oeh, dat is wel een leuke bug. :-)

denk dat een `(if-let [rio-code ...] ... (throw ...))` duidelijker is.